spark/src/test/java/org/apache/iceberg/spark/source/TestSparkDataWrite.java

```diff
@@ -67,7 +68,7 @@
 
   @Parameterized.Parameters(name = "format = {0}")
   public static Object[] parameters() {
-    return new Object[] { "parquet", "avro", "orc" };
+    return new Object[] { "parquet", "avro", "orc"};
```

Comment:
Nit: can you revert the whitespace change?

Reply:
done